format/puffin-spec.md

```diff
@@ -123,6 +123,54 @@ The blob metadata for this blob may include following properties:
 
 - `ndv`: estimate of number of distinct values, derived from the sketch.
 
+#### `delete-vector-v1` blob type
```

Comment:
We need to mention that these are uncompressed.

Reply:
Fixed.